content/participants/ttcj@vin18/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Vinit Raut
+type: "participant"
+name: "Vinit Raut"
+portfolio: "null"
+github-link: "https://github.com/vin18"
+linkedin-link: "https://www.linkedin.com/in/vinit-raut-404651148/"
+looking-for: "job"
+description: "Very good in HTML and CSS. I have good foundational understanding of React and Node.js. I have developed small projects using these stacks."
+---
+
+Hi! I am a Vinit Raut studying at Vidyalankar Institute of Technology, Mumbai. Let me walk you through my journey so far.
+
+### introduction
+
+I am a final year computer engineering student passionate about JavaScript and its ecosystems. I have developed few projects in MERN stack.
+
+## why are you taking this challenge?
+
+I need to learn and make good projects, write blogs and get recommendations from industrial experts, getting feedbacks and improve more with an wonderfull opportunity to get a job. This is a win win situation.
+
+## something interesting about you?
+
+I am 21 and I don't know how to ride a bike although have a strict 'reel world' gaming driving experience.
+
+### help I need?
+
+Usage and best practices of using GraphQl in the project. 
+
+### how to contact me?
+
+- [Twitter] (https://twitter.com/vinitraut18)
+- [LinkedIn] (https://www.linkedin.com/in/vinit-raut-404651148/)
```

Comment:
remove the space after the **]** as it's not visible as a hyperlink

Reply:
Removed the space after ].